operations/research-and-development/engineering/cloud-infrastructure-cost-kpis.md

```diff
@@ -0,0 +1,77 @@
+# Cloud Infrastructure Cost KPIs
+
+The Cloud SRE team uses certain KPIs to monitor cloud infrastructure costs from the engineering standpoint. These KPIs are used to help the business and the Cloud SRE team to set goals for cost optimization.
+
+As of July 2021, the goals for the SRE team are to
+
+ - Decrease the Average Cost Per Workspace (APWC)
+ - Keep development and test environments costs (DM) below 20% of total costs
+ - Increase the WB fraction (ratio of paid workspaces relative to total workspaces)
```

Comment:
Minor: Here we use WB and then WBM. Lets choose one of them as name and use the same. (My fault on the initial doc)

Reply:
Good catch, let's use "WB", updated